app/plugin_api.go

```diff
@@ -415,6 +415,44 @@ func (api *PluginAPI) SearchPostsInTeam(teamId string, paramsList []*model.Searc
 	return postList.ToSlice(), nil
 }
 
+func (api *PluginAPI) SearchPostsInTeamForUser(teamId string, userId string, searchParams *model.SearchParameter) (*model.PostSearchResults, *model.AppError) {
+
```

Comment:
Nit:
```suggestion
func (api *PluginAPI) SearchPostsInTeamForUser(teamId string, userId string, searchParams *model.SearchParameter) (*model.PostSearchResults, *model.AppError) {
```

Reply:
What is the suggested change? I see them the same 🤔

Reply:
See https://github.com/mattermost/mattermost-server/pull/14807#discussion_r440650228